composer.json

```diff
@@ -12,10 +12,10 @@
         "react/socket-client": "^0.5.0",
         "mkraemer/react-pcntl": "2.0.*",
         "monolog/monolog": "^1.3",
-        "paragonie/random_compat": "^2"
+        "paragonie/random_compat": "^2",
+        "guzzlehttp/psr7": "1.2.3"
```

Comment:
I took version installed by previous `composer install` command to be sure that it's ok.